service/src/main/java/bio/terra/pipelines/common/utils/FlightUtils.java

```diff
@@ -158,4 +158,10 @@ public static boolean flightComplete(FlightState flightState) {
         || flightState.getFlightStatus() == FlightStatus.FATAL
         || flightState.getFlightStatus() == FlightStatus.SUCCESS);
   }
+
+  /** Check whether the inputParameters contain a particular key and whether its value is true */
+  public static boolean inputParametersContainTrue(FlightMap inputParameters, String key) {
+    return inputParameters.containsKey(key)
+        && Boolean.TRUE.equals(inputParameters.get(key, Boolean.class));
```

Comment:
Although, sometime stairway gets picky about actually defining variables. Something like this might work instead:
```suggestion
    Boolean booleanKey = inputParameters.get(key, Boolean.class);
    return inputParameters.containsKey(key) && booleanKey;
```
You wouldn't think that pre-defining it in a variable would help, but I've come across a few cases where it is needed.

Reply:
ugh yeah this is super annoying, I don't know if TDR uses SonarQube, but it complains about this approach: 
<img width="965" alt="image" src="https://github.com/user-attachments/assets/a3d8fbcb-1b68-4d64-b880-cb2f6f5b9495">
because a `Boolean` can be null. I couldn't figure out how to extract a `boolean` from the inputParameters.

I think that we could get around this by saying if it's null, assume that's false... I'll do some poking... we hate the Boolean.TRUE.equals() thing too 😭